### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by olliebrownlow**

### CODE REVIEW

File: src/web/index.html [hunk 1]
```diff
@@ -1,30 +1,35 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <link rel="stylesheet" href="./css/style.css">
```
Comment (breycarr) [inline src/web/index.html:4]: Very neat, looks good!

File: src/web/index.html [hunk 2]
```diff
@@ -1,30 +1,35 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Plant Progress</title>
     <script type='text/javascript' src='eel.js'></script>
+
     <script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
     <script type='text/javascript' src='main.js'></script>
 
 </head>
-<body bgcolor="black" id='body'>
+<body>
```
Comment (breycarr) [inline src/web/index.html:14]: Why are there two opening body tags?

Comment (laurencetaylor) [inline src/web/index.html:14]: 2 bodies - might need changing. Also prob don't need id='body', can refer to body element itself in CSS

File: src/web/main.js [hunk 3]
```diff
@@ -108,8 +107,11 @@ async function createHistGraph() {
     }
   ];
 
-  Plotly.newPlot('all-time-chart', data);
-
+  Plotly.newPlot('all-time-chart', data, {
```
Comment (breycarr) [inline src/web/main.js:71]: Should look good!

**Review by breycarr — APPROVED** [review]
Looking forward to seeing it live and deployed on the touchscreen!

**Review by laurencetaylor — CHANGES_REQUESTED** [review]

File: src/web/index.html (hunk 2 above)
Comment (t-ekla) [reply src/web/index.html:14]: Good spot, changed.

File: src/web/main.js (hunk 3 above)
Comment (t-ekla) [reply src/web/main.js:71]: good spot, changed

**Review by laurencetaylor — APPROVED** [review]
All good, and ready to merge now

**Pull request merged**